manifest.excel.xml

```diff
@@ -80,7 +80,7 @@
       </bt:ShortStrings>
       <bt:LongStrings>
         <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```

Comment:
Actually, it should be two words "Task Pane". It was wrong for years. It looks like at some point it was fixed and now this edit is reintroducing the mistake.

```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```